packages/components/src/tabs/test/index.tsx

```diff
@@ -1773,51 +1277,33 @@ describe( 'Tabs', () => {
 			} );
 			it( 'should not render any tab when the tab associated to `initialTabId` becomes disabled while being the active tab', async () => {
 				const { rerender } = render(
-					<ControlledTabs initialTabId="gamma" selectedTabId="gamma">
-						<Tabs.TabList>
-							{ TABS.map( ( tab ) => (
-								<Tabs.Tab
-									key={ tab.id }
-									id={ tab.id }
-									title={ tab.title }
-									className={ tab.tab.className }
-								>
-									{ tab.title }
-								</Tabs.Tab>
-							) ) }
-						</Tabs.TabList>
-						{ TABS.map( ( tab ) => (
-							<Tabs.TabPanel key={ tab.id } id={ tab.id }>
-								{ tab.content }
-							</Tabs.TabPanel>
-						) ) }
-					</ControlledTabs>
+					<ControlledTabs
+						tabs={ TABS }
+						initialTabId="gamma"
```

Comment:
Same here, `initialTabId` is only meant to be used in uncontrolled mode. In controlled mode (ie. when `selectedTabId` is defined), `initialTabId` should be always ignored